tests/test_completion.py

```diff
@@ -155,7 +155,7 @@ def test_completion_plugin(
     @pytest.mark.parametrize(  # type: ignore
         "prog", [["python", "hydra/test_utils/completion.py"]]
     )
-    @pytest.mark.parametrize("shell", ["bash", "fish"])  # type: ignore
+    @pytest.mark.parametrize("shell", ["bash", "fish", "zsh"])  # type: ignore
```

Comment:
can you ensure that the zsh tests are skipped if zsh is not installed locally?
I think there should be something similar in place for fish.

Reply:
Added the check and squashed everything into one commit.